include/render/wlr_renderer.h

```diff
@@ -21,5 +21,11 @@ bool wlr_renderer_bind_buffer(struct wlr_renderer *r, struct wlr_buffer *buffer)
  */
 const struct wlr_drm_format_set *wlr_renderer_get_render_formats(
 	struct wlr_renderer *renderer);
+/**
+ * Get the supported buffer capabilities.
+ *
+ * This functions returns a bitfield of supported wlr_buffer_cap.
+ */
+uint32_t renderer_get_buffer_caps(struct wlr_renderer *renderer);
```

Comment:
I'd prefer to name this `renderer_get_render_buffer_caps`, to make it clear this queries information about buffers that will be rendered to, not buffers which are textured from.